src/third-party-tags/inizio.ts

```diff
@@ -0,0 +1,34 @@
+import { GetThirdPartyTag } from '../types';
+
+const onLoad = () => {
+	const handleQuerySurveyDone = (
+		surveyAvailable: boolean,
+		survey: { measurementId: string },
+	) => {
+		if (surveyAvailable) {
+			if (window && window.googletag) {
+				window.googletag.cmd.push(() => {
+					window.googletag.pubads().setTargeting('inizio', 't');
+				});
+			}
+			console.log(`surveyAvailable: ${survey.measurementId}`);
+		}
+	};
+	// eslint-disable-next-line no-underscore-dangle
+	window._brandmetrics = window._brandmetrics || [];
```

Comment:
if you wanna be hipster 🕺 
```suggestion
	window._brandmetrics ||= [];
```

Reply:
If I **commit suggestion,** will it make commitizen unhappy?

Reply:
ooh dont know – wanna try it?